.circleci/config.yml

```diff
@@ -293,11 +293,15 @@ jobs:
       goarch:
         type: string
         default: "amd64"
+      enable_race_testing:
+        type: string
```

Comment:
for the circleci bit, probably. however, downstream usage would require that it be converted to an empty string when `false`, because of the way that conditionals work in GNU make:
https://github.com/hashicorp/nomad/pull/10302/files#diff-78a8a19706dbd2a4425dd72bdab0502ed7a2cef16365ab7030a5a0588927bf47R304

Reply:
okay, i've changed my mind. will do that, at least for the purpose of documentation.

Reply:
@notnoop , when you have a chance, i'd appreciate your feedback on this approach.

Reply:
specifically, this: 
https://github.com/hashicorp/nomad/pull/10302/files#diff-78a8a19706dbd2a4425dd72bdab0502ed7a2cef16365ab7030a5a0588927bf47R304